src/mon/PGMap.cc

```diff
@@ -4550,8 +4556,41 @@ int reweight::by_utilization(
       osd_util.second = (double)p.second.kb_used / (double)p.second.kb;
     }
     util_by_osd.push_back(osd_util);
+
+    // Overweighted devices and their weights are factors to
```

Comment:
i believe Ceph uses https://google.github.io/styleguide/cppguide.html#Comment_Style instead of the one on wikibooks.